### DISCUSSION TITLE
[Blog] O byciu architektem

### DISCUSSION THREAD

**Discussion by wi5nia** [AzureCommunityPL/group#62] [Community Content]
Kilka przemyslen, uwag, spostrzezen na temat bycia i nie bycia architektem 😃 

![](https://tomaszwisniewski.com/assets/images/architectornot/aorna.png)

🌐 Blog: [https://tomaszwisniewski.com/thoughts/2020/09/10/architect-or-not-an-architect.html](https://tomaszwisniewski.com/thoughts/2020/09/10/architect-or-not-an-architect.html)

**Comment by piotrgwiazda** [comment]
Ale super post!!! Trochę krótki ale w punkt.

czw., 10 wrz 2020, 11:37 użytkownik Tomasz Wisniewski <
notifications@github.com> napisał:

**Reply by wi5nia** [reply]
Takie czasy, że jak blog ma więcej nietechnicznych niż 5 minut czytania to większość odpada ;)
No i dzieki!
